ccmlib/node.py

```diff
@@ -259,8 +259,12 @@ def address_and_port(self):
         """
         Returns the IP used for internal communication along with ports
         """
+        address = self.network_interfaces['storage'][0]
         port = self.network_interfaces['storage'][1]
-        return self.network_interfaces['storage'][0] + ':' + str(port)
+        if address.find(":") != -1:
+            return "[{}]:{}".format(address, port)
```

Comment:
didn't test on ipv6 so not sure if something else will fail to handle the ipv6 address format...